src/newrelic.ts

```diff
@@ -178,6 +186,7 @@ function newrelic(this: any, options: NewRelicOptions) {
 
 // Default options.
 const defaults: NewRelicOptions = {
+  enabled: false,
```

Comment:
Please write a unit test that tests that the `enabled` option defaults to `false`. The test can be written here in the `test/newrelic.test.ts` file.

The test would look something like this - but not necessarily same:
```javascript
test('the plugin is disabled by default', () => {
  expect(Plugin.defaults).toEqual(
    expect.objectContaining({
      enabled: false
    })
  )
})
```